src/sentry/static/sentry/app/components/organizations/headerItem.tsx

```diff
@@ -54,9 +58,10 @@ class HeaderItem extends React.Component<Props> {
       locked,
       lockedMessage,
       settingsLink,
-      onClear, // eslint-disable-line no-unused-vars
+      onClick,
       loading,
-      ...props
```

Comment:
Any concern dropping spread props?

Reply:
Looks like I missed `data-test-id`, I'll see if I can make it work but ts was blowing up with the spread.

Reply:
K reverted and changed `HTMLProps` to `HTMLAttributes`